app/config/config_test.yml

```diff
@@ -39,6 +39,14 @@ sylius_rbac:
 sylius_channel:
     fake_channel_support: true
 
+sylius_theme:
+    sources:
+        filesystem:
+            locations:
+                - "%kernel.root_dir%/themes"
+                - "%kernel.root_dir%/../vendor/sylius/themes"
+                - "%kernel.cache_dir%/_themes"
```

Comment:
Will be rewritten, it's not the best idea for persisting themes through Behat scenario.